api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -45,6 +45,8 @@
 
     public static final Boolean DEFAULT_ALLOW_REMOTE_LIST_DOWNLOAD = false;
 
+    public static final int DEFAULT_REMOTE_METADATA_TIMEOUT_SECONDS = 86400;
```

Comment:
I think we agreed on this previously, but I wonder if we should set it to 43200 (1/2 day).

Reply:
As this is configurable, if we want 1/2 day we can set it anytime through main.conf